<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notenliste</class>
 <widget class="QDialog" name="Notenliste">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>431</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="kurse_layout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>427</width>
         <height>457</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="statistik_layout"/>
  </widget>
  <widget class="QLabel" name="notenuebersicht_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Notenübersicht</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatistikLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Statistik</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="kurshinzufuegenLayout">
    <item>
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="bewertungLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ectsLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="hinzufuegenButton">
      <property name="text">
       <string>Kurs hinzufügen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Kurse Erstellen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="vergleichButton">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>580</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
